pkgs/servers/web-apps/vikunja/frontend.nix

```diff
@@ -1,26 +1,13 @@
-{ stdenv, lib, fetchurl, unzip, nixosTests, ... }:
+{ lib, fetchzip, nixosTests, ... }:
 
-stdenv.mkDerivation rec {
+fetchzip rec {
   pname = "vikunja-frontend";
   version = "0.22.0";
 
-  src = fetchurl {
-    url = "https://dl.vikunja.io/frontend/${pname}-${version}.zip";
-    hash = "sha256-LYU1IGAhSZOwMI44HW5IfcBKs9RkU/IXGVgPGDKnKAs=";
-  };
-
-  nativeBuildInputs = [ unzip ];
-
-  sourceRoot = ".";
-
-  installPhase = ''
-    runHook preInstall
-
-    mkdir -p $out/
-    cp -r * $out/
+  url = "https://dl.vikunja.io/frontend/${pname}-${version}.zip";
+  hash = "sha256-mMfK+58nN014ta09/mTbz/6BgQIBj6DqJSPaRtGZ3A8=";
```

Comment:
Why is the hash different even if the url didn't change?

Reply:
the prior hash was using fetchurl, which stored the archive. This one hashes the unpacked results

Reply:
Makes sense, thanks for the clarification.